src/Scanner.js

```diff
@@ -167,6 +181,8 @@ class DefaultScanner {
         clearTimeout(this.debounceTimer)
         this.changedElements.clear()
         this.mutObs.disconnect()
+        window.removeEventListener('pointerdown', this, true)
+        window.removeEventListener('focus', this, true)
```

Comment:
No need to check if we're on mobile. If the listener wasn't attached, the method is a noop.